apps/private-law/automation/kustomization.yaml

```diff
@@ -20,3 +20,6 @@ resources:
   - ../prl-ccd-case-migration/image-repo.yaml
   - ../prl-ccd-case-migration/demo-image-policy.yaml
   - ../prl-ccd-case-migration/image-policy.yaml
+  - ../prl-cron-fm5-reminder/demo-image-policy.yaml
```

Comment:
Would suggest reviewing this image policy file:
```
/tmp/ptl-intsvc/00/ImagePolicy-demo-prl-cron-fm5-reminder.yml - ImagePolicy demo-prl-cron-fm5-reminder is invalid: problem validating schema. Check JSON formatting: jsonschema: '/spec' does not validate with file:///tmp/schemas/ptl-intsvc/00/master-standalone-strict/imagepolicy-image-v1beta1.json#/properties/spec/required: missing properties: 'policy'
```

Reply:
Thanks @reespozzi, it's fixed